### PR TITLE
Online Parser, YAML data validation, NetTowel

### PR DESCRIPTION
**Pull request by ubaumann**
- Created a new section online parser and moved the Jinja online parser to the new location. Also added textFSM and TTP parser I know of.
- Blog post on YAML data validation added to the YAML section as I think more people should know about this topic
- NetTowel Cli Tool added to the Tools section

If I should change anything or if you disagree with me, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +149,7 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
  - [Sample YAML documents describing common data structures](https://github.com/ipspace/NetOpsWorkshop/tree/master/YAML) - YAML examples courtesy of Ivan Pepelnjak.
  - [YAML syntax validator](https://yamllint.readthedocs.io/en/latest/) - YAML online parser.
  - [YAML idiosyncrasies](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html) -- Document with the most common YAML idiosyncrasies, hosted under the SaltStack docs, but not specific to Salt or any other tool.
+ - [YAML Data Validation with JSON Schema](https://infrastructureascode.ch/yaml_validation.html) - Explains how JSON schema can be used to validate YAML files and how syntax highlighting can be added to the IDE for the YAML files
```
Comment (itdependsnetworks) [inline README.md:152]: Can you add a period?

File: README.md [hunk 2]
```diff
@@ -338,13 +338,34 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
 
 - [InfluxDB](https://www.influxdata.com/) - Made for developers to build time-series-based applications quickly and at scale.
 - [Prometheus](https://prometheus.io/) - Prometheus is an open-source systems monitoring and alerting toolkit originally built at SoundCloud.
 - [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) - elegraf is the open source server agent to help you collect metrics from your stacks, sensors, and systems.
 
+## Online Parser
+
+### Jinja2
+
+ - [J2Live - TTL255](https://j2live.ttl255.com/) - Online Jinja2 parser. Additional support for Ansible and Salt filters.
```
Comment (itdependsnetworks) [inline README.md:353]: Make this (and everything else) in alphabetical order.

File: README.md [hunk 3]
```diff
@@ -338,13 +338,34 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
+ - [TD4a](https://td4a.codethenetwork.com/) - Advanced online rendering tool.
+ - [textfsm.nornir.tech](https://textfsm.nornir.tech/) - Online Jinja2 parser.
+ - [Jinja 101](https://jinja101.infrastructureascode.ch/) - Online Jinja2 parser with Ansible, Staltstack and Stackstorm filter support. Jinja2 environment settings can be configured.
+
+### TextFSM
+
+ - [textfsm.nornir.tech](https://textfsm.nornir.tech/) - Online TextFSM parser.
```
Comment (itdependsnetworks) [inline README.md:360]: Ideally we can flush these types of sentences out a bit better.

File: README.md (hunk 1 above)
Comment (ubaumann) [reply README.md]: Of cource. Thanks for seeing it.

File: README.md (hunk 2 above)
Comment (ubaumann) [reply README.md:353]: I sorted the Online Parser section

File: README.md (hunk 3 above)
Comment (ubaumann) [reply README.md]: I am not a great text writer but I tried something :)

File: README.md [hunk 4]
```diff
@@ -338,13 +338,34 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
 - [NetCopa](https://github.com/cidrblock/netcopa) - Network device configuration parser ("industry standard" -> YAML converter).
 - [OSPF Watcher](https://github.com/Vadims06/ospfwatcher) - Tracks OSPF topology changes--by establishing a GRE tunnel with network devices--via a history diagram.
 - [Topolograph](https://github.com/Vadims06/topolograph) - Python-based Web tool for visualisation of OSPF/ISIS topologies and making a prediction of network behaviour in case of network's outage.
+- [NetTowel](https://github.com/InfrastructureAsCode-ch/nettowel) - Collection of useful network automation functions for the CLI.
```
Comment (itdependsnetworks) [inline README.md:341]: sort please.

File: README.md [hunk 5]
```diff
@@ -148,6 +149,7 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
  - [Sample YAML documents describing common data structures](https://github.com/ipspace/NetOpsWorkshop/tree/master/YAML) - YAML examples courtesy of Ivan Pepelnjak.
  - [YAML syntax validator](https://yamllint.readthedocs.io/en/latest/) - YAML online parser.
  - [YAML idiosyncrasies](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html) -- Document with the most common YAML idiosyncrasies, hosted under the SaltStack docs, but not specific to Salt or any other tool.
+ - [YAML Data Validation with JSON Schema](https://infrastructureascode.ch/yaml_validation.html) - Explains how JSON schema can be used to validate YAML files and how syntax highlighting can be added to the IDE for the YAML files.
```
Comment (itdependsnetworks) [inline README.md:152]: sort please.

File: README.md [hunk 6]
```diff
@@ -338,13 +338,34 @@ Network Automation is a cross between the discipline of [Network Infrastructure]
...
 - [InfluxDB](https://www.influxdata.com/) - Made for developers to build time-series-based applications quickly and at scale.
 - [Prometheus](https://prometheus.io/) - Prometheus is an open-source systems monitoring and alerting toolkit originally built at SoundCloud.
 - [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) - elegraf is the open source server agent to help you collect metrics from your stacks, sensors, and systems.
 
+## Online Parser
+
+### Jinja2
+
+ - [J2Live - TTL255](https://j2live.ttl255.com/) - Online Jinja2 parser. Additional support for Ansible and Salt filters.
+ - [Jinja 101](https://jinja101.infrastructureascode.ch/) - Online Jinja2 parser with Ansible, Staltstack and Stackstorm filter support. Jinja2 environment settings can be configured.
```
Comment (itdependsnetworks) [inline README.md:354]: Staltstack -> SaltStack

**Pull request merged**